src/Components/WebAssembly/Build/src/targets/Publish.targets

```diff
@@ -18,13 +18,14 @@
       <!-- Delete stray contents from the root of the the app. -->
       <ResolvedFileToPublish
         Remove="%(ResolvedFileToPublish.Identity)"
-        Condition="!$([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').Replace('\','/').StartsWith('wwwroot/'))"/>
+        Condition="'%(ResolvedFileToPublish.RelativePath)' != 'web.config' AND !$([System.String]::Copy('%(ResolvedFileToPublish.RelativePath)').Replace('\','/').StartsWith('wwwroot/'))"/>
```

Comment:
Perhaps we can use a regex or add another check for `web.Release.config`? Ditto for below.

Reply:
Nevermind. `web.Release.config` files are handled differently and don't need to be copied to the publish directory.